Cargo.toml

```
@@ -9,7 +9,6 @@ readme       = "README.md"
 categories   = ["cryptography::cryptocurrencies", "encoding", "no-std"]
 keywords     = ["blockchain", "cosmos", "tendermint", "ibc", "proto"]
 exclude      = ["definitions", "tools"]
-rust-version = "1.60"
```

I assumed it's okay to do this

---

Apparently cargo uses it to fail early. I think this should normally match the MSRV but it's optional.